src/main/java/seedu/address/model/task/Entry.java

```diff
@@ -160,7 +160,8 @@ public String getDateDisplay(LocalDateTime dateTime){
         if (dateTime == null) {
             return "";
         }
-        return dateTime.format(DATE_TIME_FORMATTER);
+        Date interpreted = Date.from(dateTime.atZone(ZoneId.systemDefault()).toInstant());
+        return prettyTime.format(interpreted);
```

Comment:
I think the pretty time format looks great for deadlines, maybe not quite for events though?